### ISSUE TITLE
DAX studio trims unprintable characters from values on export from Power Pivot model to SQL Server using Export Data Wizard

### ISSUE THREAD

**Issue by bogomda** [DaxStudio/DaxStudio#526]
I export data from Power Pivot Model to SQL Server using Export Data Wizard and it works great except one nuance.

In Power Pivot Excel source there are values with not visible space characters (" ") created on purpose. For example, you see **EXPN: Expenses** value four time even though they look same. 
When data gets transferred to SQL Server database, the spaces gets trimmed off and there is only one **EXPN: Expenses** value on the target side.
Is it something can be fixed?
![Data Export](https://user-images.githubusercontent.com/66798479/105734198-612e6000-5f00-11eb-906c-d18b4daafc7d.PNG)

**Comment by dgosbell** [comment]
We don't do any processing of the values during export. We basically run an EVALUATE TOPNSKIP( 1000000, 0 , <table name> ) and stream the results into SQL using a bulk insert. I think what you might be seeing is the DISTINCT operator in SQL [ignoring trailing whitespace](https://stackoverflow.com/questions/38288146/sql-server-distinct-with-trailing-whitespace). If you do a rowcount does the sql table and the tabular table have the same number of total rows

**Comment by bogomda** [comment]
Darren,
yes, both Tabular and SQL Server tables have same row count, I didn't find it to be a problem.

While it's important to know such behavior on the target side (SQL ignores trailing whitespaces), I found the resolution to avoid the problem:
on Tabular side (source), I used zero length white space (ZLWS) character in stead of normal space " ". SQL server doesn't trim trailing ZLWS characters and values come from Tabular to SQL table as expected, even though visually values look exactly same.

In Power BI, it's more straight forward to use ZLWS character with delivered formula - UNICHAR ( 8204 ).
In Excel there is no such formula, and I had to copy ZLWS character from https://codepen.io/chriscoyier/pen/iLKwm and paste inside the quotes "​" in DAX formula to make it work. These are not empty quotes, there is a ZLWS inside of them.

Thanks so much for such a brilliant tool.
